etc/FORCH_VERSION

```diff
@@ -1 +1 @@
-0.48
+0.48.4
```

Comment:
0.48.4 is a gmaster (internal only) tag .. should likely be 0.49  -- if you look at 0.48.3 it's a descendant of gmaster, not master.